### DISCUSSION TITLE
Where is the Material Bottom Tabs package?

### DISCUSSION THREAD

**Discussion by panoramix360** [react-navigation/react-navigation#11273] [Q&A] [answered]
Hello everyone! I hope you are well.

I'm updating versions of a project that was paused 2 months ago.

I was looking through the packages to see the change logs of the new version of the `@react-navigation/material-bottom-tabs` package that the project is using and I couldn't find anymore.

Where is the new location of it or is it moved to another repo?

I looked through the [documentation](https://reactnavigation.org/docs/material-bottom-tab-navigator/), and the link is broken there as well.

Thank you!

**Comment by liamjones** [comment, marked as answer]
>  chore: drop material-bottom-tabs package
>
> BREAKING CHANGE: the navigator should be imported from 'react-native-paper/react-navigation' instead when it's available

https://github.com/react-navigation/react-navigation/commit/83d4f8682ce5e6342022844630dfaae84b14a98f

**Reply by panoramix360** [reply]
Thank you @liamjones, but that means the Changelog and stuff like that will be on React Native Paper repo now, right?

**Reply by liamjones** [reply]
I would guess so but I don't know for sure, I don't use the material-bottom-tabs package myself.
